bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/Client.scala

```diff
@@ -151,16 +142,34 @@ trait Client
       case _: PasswordBased => Future.successful(())
 
     }
+    val isAlreadyStarted: Future[Boolean] = isStartedF
+
+    val started: Future[Future[BitcoindRpcClient]] = isAlreadyStarted map {
+      case false =>
+        if (version != BitcoindVersion.Unknown) {
+          val foundVersion = instance.getVersion
```

Comment:
I don't think we can get the version when it is `isRemote` (if bitcoind isn't installed)